en_US.ini

```diff
@@ -135,7 +135,7 @@ Log Console = &Log Console
 Memory View... = Memory &View...
 More Settings... = &More Settings...
 Nearest = &Nearest
-Non-Buffered Rendering = &Non-Buffered Rendering (speedhack)
+Non-Buffered Rendering = &Discard Effects (Non-Buffered, speedup)
```

Comment:
Hmm, that's hard because Non-Buffered Rendering may or may not discard effects, also it does more than just that.